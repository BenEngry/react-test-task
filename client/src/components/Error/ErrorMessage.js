import React from "react";

export const ErrorMessage = (error) => {
    return (
        <div className="wrapper">
            <span>Error!</span>
        </div>
    ) 
};