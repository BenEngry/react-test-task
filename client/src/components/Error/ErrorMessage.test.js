import renderer from "react-test-renderer";
import React from "react";
import { ErrorMessage } from "./ErrorMessage";


describe("Testing <ErrorMessage/>", ()=> {

    it("List have rendered correctly", ()=>{
        const errorComp = renderer.create(<ErrorMessage error={"Error example"}/>).toJSON();
        expect(errorComp).toMatchSnapshot();
    });

    it("should render a error name", ()=>{
        const errorExample = "Error example";
        const errorComp = renderer.create(<ErrorMessage error={errorExample}/>);
        expect(errorComp.prop("example")).toEqual("Error example");
    });

});
