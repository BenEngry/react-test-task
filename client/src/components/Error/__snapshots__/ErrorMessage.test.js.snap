// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <ErrorMessage/> List have rendered correctly 1`] = `
<div
  className="wrapper"
>
  <span>
    Error!
  </span>
</div>
`;

exports[`Testing <ErrorMessage/> should render a error name 1`] = `
<div
  className="wrapper"
>
  <span>
    Error!
  </span>
</div>
`;
